app/src/main/java/com/sopt/kakaobank/presentation/history/HistoryAdapter.kt

```diff
@@ -0,0 +1,63 @@
+package com.sopt.kakaobank.presentation.history
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.sopt.kakaobank.data.dto.response.History
+import com.sopt.kakaobank.databinding.ItemHistoryDepositBinding
+import com.sopt.kakaobank.databinding.ItemHistoryWithdrawBinding
+
+class HistoryAdapter() : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
+
+    // 임시의 빈 리스트
+    private var historyList: List<History> = emptyList()
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RecyclerView.ViewHolder {
+        return when (viewType) {
+            // 입금
+            TYPE_DEPOSIT -> {
+                val binding = ItemHistoryDepositBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+                DepositViewHolder(binding)
+            }
+            // 출금
+            TYPE_WITHDRAW -> {
+                val binding = ItemHistoryWithdrawBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+                WithdrawViewHolder(binding)
+            }
+            else -> throw IllegalArgumentException("Invalid view type")
+        }
+    }
+
+    override fun onBindViewHolder(holder: RecyclerView.ViewHolder, position: Int) {
+        val profile = historyList[position]
+
+        when (holder) {
+            is DepositViewHolder -> {
+                holder.onBind(profile)
+            }
```

Comment:
```suggestion
           is DepositViewHolder -> holder.onBind(profile)
```
전 간결한 코드를 선호해서 ㅎ.ㅎ 별 차이는 없어용